.travis.yml

```diff
@@ -1,28 +1,32 @@
 language: objective-c
-osx_image: xcode10
+os: osx
+osx_image: xcode10.1
 xcode_project: YMCache.xcodeproj
-xcode_scheme: YMCache-iOS
+
+matrix:
+  include:
+    - xcode_scheme: YMCache-iOS
+      xcode_destination: platform=iOS Simulator,OS=12.1,name=iPhone X
+
+    - xcode_scheme: YMCache-Mac
+      xcode_destination: platform=macOS
+
 before_install:
-- brew update || brew update
-- brew install carthage || brew outdated carthage || brew upgrade carthage
-- gem install xcpretty --no-rdoc --no-ri --no-document --quiet
-- gem install xcpretty-travis-formatter --no-rdoc --no-ri --no-document --quiet
-- gem install slather --no-rdoc --no-ri --no-document --quiet
-- "./script/travis-codesign-setup.sh"
+  - gem install slather --no-rdoc --no-ri --no-document --quiet
+  - brew update || brew update
```

Comment:
why twice?

Reply:
it was some issue where the first one emits a non-zero error code if the update occurred or something. annoying.